### ISSUE TITLE
User is not able to resize column headers using single pointer mode of operation: A11y_Visual Studio Code Jupyter Extensions_Dragging movement.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-jupyter#17054]
### GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #A11ySev2;#DesktopApp; #Visual Studio Code Jupyter Extensions; #GH-VSCodeJupyterExtensions-Win32-Oct2025; #WCAG2.5.7; #FTP;#Dragging Movements;

### Environment Details:
Application: Visual Studio Code Jupyter Extensions
Version: 1.105.0-insider (user setup)
OS build :26100.4770

### Repro Steps

1. Open vs code.
2. Install Jupyter Extension.
3. Run and execute cell.
4. Juypter window will open with table.
5. Column headers are not accessible using the keyboard.

### Actual Experience.
User is not able to resize column headers using single pointer mode of operation.

### Expected Experience
User should be able to resize column header using single pointer mode of operation without dragging.

### User Impact:
Functionality provided only with dragging will create difficulty for users using input devices like a trackball, head pointer, eye-gaze system to perform the operation.

### Attachment

https://github.com/user-attachments/assets/ef6eac53-db41-45ee-ad08-6282942daa1e

**Comment by amunger** [comment]
> Juypter window will open with table.

What code are you running in the cell to make this happen?

**Comment by amunger** [comment]
I see now that this is the jupyter variable explorer, that view will hopefully just be replaced by the native treeview (enabled with the `notebook.variablesView` setting).

**Comment by amunger** [comment]
Closing as we are deprecating this view in favor of the native variable explorer with https://github.com/microsoft/vscode/issues/166296, available with the `notebook.variablesView` setting enabled
